debian/postrm

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+
+#DEBHELPER#
+
+case $1 in
+    remove)
+        deluser -r oem || true
```

Comment:
1. What if you run `systemd-sysusers` again? Would it update the system users again based on the configuration file?
2. If the account won't be deleted by `systemd-sysusers`, then maybe `usermod -L -e 1 oem` instead of delete it.

Reply:
1. Running `systemd-sysusers` again will do `nothing` i.e. wont remove the user but it wont add any new users even if the config is removed